test/query-test.js

```diff
@@ -7,7 +7,7 @@ import variantConnectionQuery from '../src/variant-connection-query';
 import optionQuery from '../src/option-query';
 import imageQuery from '../src/image-query';
 import imageConnectionQuery from '../src/image-connection-query';
-import collectionQuery from '../src/collection-query';
+import collectionNodeQuery from '../src/collection-node-query';
```

Comment:
It uses `collectionConnectionQuery` to fetch multiple collections off shop and `collectionNodeQuery` to fetch a collection off the root.